apps/climatemappedafrica/src/lib/data/blockify/hero.js

```diff
@@ -0,0 +1,34 @@
+import {
+  fetchProfile,
+  fetchProfileGeography,
+} from "@/climatemappedafrica/lib/hurumap";
+
+export default async function hero(block) {
+  const { geometries } = await fetchProfileGeography(
+    block.location?.name?.toLowerCase() ?? "af",
+  );
+  const { level } = geometries.boundary?.properties ?? {};
+  const childLevelMaps = {
+    continent: "country",
+    country: "region",
+  };
```

Comment:
Just thinking out loud, doesn't the geographies/geometries we get from HURUmap define hierarchy we can use rather than manually computing it.

Reply:
@kilemensi for KE this is the object response from hurumap
<img width="426" alt="image" src="https://github.com/user-attachments/assets/22e91d96-61b1-43fe-b87d-c7814c014cde">
Any other country, it uses the Region key, see below (TZ, DJ, etc)
<img width="566" alt="image" src="https://github.com/user-attachments/assets/ed331ee6-3d5b-45c9-8baa-8ba3d52fbda4">
for continent,
<img width="449" alt="image" src="https://github.com/user-attachments/assets/cb7554aa-d462-4583-851d-dfbdd2912d5b">
hence the decision

Reply:
@koechkevin I agree with @kilemensi that we do not need to compute it manually, other countries may have `Province`, for example, Angola. We had a similar problem with @thepsalmist, and we resolved it by adding every possible country-level subdivision to the `preferred_children` array in the profile config on HURUMap, then access it as follows: 

[`const preferredChildrenPerLevel = preferredChildren[level];`](https://github.com/CodeForAfrica/ui/blob/62f1e858338f21e6643a8a8e60fc0eeade2a5ea8/packages/hurumap-next/src/Map/LazyMap.js#L41). 

We can have a call to clarify it if need be